### PR TITLE
add log based tests for verifying WPC GM state

### PR DESCRIPTION
**Pull request by nishant-parekh**

### CODE REVIEW

**Review by aneeshkp — APPROVED** [review]

File: test/conformance/serial/ptp.go [hunk 1]
```diff
@@ -851,6 +851,81 @@ var _ = Describe("["+strings.ToLower(DesiredMode.String())+"-serial]", Serial, f
 					Skip("Failed to find a valid ptp slave configuration")
 				}
 			})
+			FIt("is verifying WPC GM state based on logs", func() {
```
Comment (jzding) [inline test/conformance/serial/ptp.go:854]: Fit => It

**Pull request merged**